hw/ip/edn/doc/_index.md

```diff
@@ -199,18 +199,18 @@ See the [CSRNG IP]({{< relref "hw/ip/csrng/doc" >}}) waveform section for the CS
 ##### Peripheral Hardware Interface - Req/Ack
 The following waveform shows an example of how the peripheral hardware interface works.
 This example shows the case where the boot-time mode in the ENTROPY_SRC block is enabled.
+This example also shows the case where the next request will change the pior data by popping the data FIFO.
```

Comment:
nit: should this be "p**r**ior data"?

Reply:
It is actually a valid Portuguese word that means worse, but since that was not the intend, I will go with all English - fixed.